src/univ/notebook.cpp

```diff
@@ -688,7 +674,7 @@ wxRect wxNotebook::GetTabRect(int page) const
     else
     {
         widthBefore = 0;
-        for ( int n = 0; n < page; n++ )
+        for (size_t n = 0; n < page; n++)
```

Comment:
```suggestion
        for ( size_t n = 0; n < page; n++ )
```

Reply:
Already fixed in latest commit

Reply:
This is pretty minor, but it doesn't seem to have been fixed.